src/components/Dashboard/index.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import CardContainer from '../CardContainer';
 import { withRouter } from 'react-router'
-import { Route } from 'react-router-dom';
 
 export const Dashboard = ({ location }) => {
   const dataKey = location.pathname.split('/').slice(-1)[0] || 'tickets';
   
   return (
     <main>
-      <CardContainer dataKey={dataKey} />
+      { dataKey === 'tickets'&& <CardContainer dataKey={dataKey} /> }
+      { dataKey === 'resources' && <CardContainer dataKey={dataKey} /> }
```

Comment:
I think this would work with just <CardContainer datakey={dataKey} /> without the &&. Have you tried it without?